test/utils/DateFormatter.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+
+import { dateToString } from 'app/utils/DateFormatter';
+
+describe('DateFormatter', () => {
+
+    it('should throw invalid date error', () =>{
+
+        const date: Date = new Date('nonsense date value');
+        expect(() => dateToString(date)).throws('DateFormatter - Invalid date');
+
+    });
+
+    it('should return YYYY-MM-DD format for a local date', () =>{
+
+        const date: Date = new Date('2020-10-30T23:00:00.000');
+        expect(dateToString(date)).to.equal('2020-10-30');
+
+    });
+
+    it('should return YYYY-MM-DD format for a UTC date', () =>{
+
+        const date: Date = new Date('2020-10-30T23:00:00.000z');
+        expect(dateToString(date)).to.equal('2020-10-30');
+
+    });
+
+    it('should return incremented month value [0-11] by one for correct string representation [1-12]', () =>{
```

Comment:
That sounds super low level and it only applies to dates created from individual date components. It does not look good to me.

Reply:
this `dateToString` function could be called from other locations in the future, I want to make sure to have a test that covers this case. Its a small but quite dangerous little detail

Reply:
renamed it to : 
`'should return correct month number for a given month value'`